feature/system/gnmi/cliorigin/ate_tests/mixed_oc_cli_origin_support_test/mixed_oc_cli_origin_support_test.go

```diff
@@ -124,22 +124,35 @@ func testQoSWithCLIAndOCUpdates(t *testing.T, dut *ondatra.DUTDevice, tCase test
 	}
 }
 
-func TestQoSDependentCLI(t *testing.T) {
+func TestQoSDependentCLIFullReplace(t *testing.T) {
+	// TODO: Skipping this test case until we can figure out how to signal
+	// different subtests' test results.
```

Comment:
As discussed, change to: skipping this test because it is not expected to pass right now.